rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/with-contenv sh
+
+SIDECAR_SNNMPTRAPD=${SIDECAR_SNNMPTRAPD:-0}
+
+# Continue only if sidecar snmptrapd container
+if [ "$SIDECAR_SNNMPTRAPD" != "1" ]; then
+  exit 0
+fi
+
+echo ">>"
+echo ">> Sidecar snmptrapd container detected"
+echo ">>"
+
+mkdir -p /data/snmptrapd /run/snmptrapd
+chown librenms. /data/snmptrapd
+chown -R librenms. /run/snmptrapd
+
+# Create service
+mkdir -p /etc/services.d/snmptrapd
+cat > /etc/services.d/snmptrapd/run <<EOL
+#!/usr/bin/execlineb -P
+with-contenv
+/usr/sbin/snmptrapd -f -m IF-MIB -M /opt/librenms/mibs:/opt/librenms/mibs/cisco
```

Comment:
The `-m` option should be `-m +IF-MIB` without the `+` only `IF-MIB` is included and nothing else or we could simply do `-m ALL`

Reply:
I did it this way because that's the example on https://docs.librenms.org/Extensions/SNMP-Trap-Handler/ , but I agree, saying -m ALL is better.